src/components/BookContainer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+// import { any, object } from 'prop-types';
```

Comment:
- I strongly request to delete these unused codes because we recommend clean codes. This will make your work more Professional